src/renderer/components/tree-view/tree-label-edit.js

```diff
@@ -76,3 +82,12 @@ export default class LabelEditor extends Component {
     );
   }
 }
+
+LabelEditor.propTypes = {
+  node: PropTypes.object,
```

Comment:
Node is used without checks so should be `isRequired` here.

Reply:
I know that you didn't invent the code of course, but if we're changing it then it's a good time to update it 😄

Reply:
I'll change it ;) Don't know if it's better to write `propTypes` in- or outside class. What do you prefer?

Reply:
I think `static propTypes` within classes is what we're going for now - last I spoke with @sallar :)